src/ui/handler/touch_zoom_rotate.js

```diff
@@ -251,7 +252,7 @@ export class TouchPitchHandler extends TwoTouchHandler {
         };
     }
 
-    gestureBeginsVertically(vectorA: Point, vectorB: Point, timeStamp: number) {
+    gestureBeginsVertically(vectorA: Point, vectorB: Point, timeStamp: number): void | boolean {
```

Comment:
Not sure about returning `void` here. Should we return `false` here?

https://github.com/mapbox/mapbox-gl-js/blob/ddb84f13f4dcdd7be2be997a3d690faa7a5c7c49/src/ui/handler/touch_zoom_rotate.js#L262-L263

Reply:
It seems that there's logic that depends on it being `undefined` here, so let's keep it like this.